plugins/SLURM-Integration/install.sh

```diff
@@ -64,6 +64,19 @@ function update_slurm_conf()
     fi
 }
 
+function update_python_path_for_ufm_prolog_epilog()
+{
+    python_path=$(which python3)
+    sudo sed -i -e "s#python3_path#$python_path#g" $SLURM_DIR/$UFM_PROLOG_FILE
+    if [[ $? -ne 0 ]];then
+        echo "Warning:: Failed to replace the output of 'which pythn3' command with python3_path keyword in $SLURM_DIR/$UFM_PROLOG_FILE file, please try to do it manually."
+    fi
+    sudo sed -i -e "s#python3_path#$python_path#g" $SLURM_DIR/$UFM_EPILOG_FILE
```

Comment:
same, check if really need sudo and granular error check with user facing messages

Reply:
I checked it and we need to run it with sudo.

Reply:
maybe run entire script with ```sudo install.sh``` and not specific lines?
sudo is not default everywhere
script can also check that active user is root and fail if not.

Reply:
I tried to run the entire install script with sudo as well, but it failed with permission error to the sed command "sed: couldn't edit /etc/slurm/: not a regular file", farther that we are running all the command inside the install script with sudo and if I will change it now I'm sure that we will have many issues with permissions.

Reply:
```couldn't edit /etc/slurm/:``` because actual filename is missing in the string. seems bug in a code